generate_devices_list.py

```diff
@@ -10,7 +9,7 @@
 except ImportError:
     import simplejson as json
 
-POSSIBLE_TYPES = ["phone", "tablet", "notebook", "desktop", "unknown"]
+from jsonschema import validate
```

Comment:
Please import the module as a whole, and use |jsonschema.validate| to avoid confusion.